sdk/fabric/Azure.ResourceManager.Fabric/tests/FabricManagementTestBase.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Fabric.Tests
+{
+    public class FabricManagementTestBase : ManagementRecordedTestBase<FabricManagementTestEnvironment>
+    {
+        protected ArmClient Client { get; private set; }
+        protected SubscriptionResource Subscription { get; private set; }
+        protected ResourceGroupResource ResourceGroup { get; private set; }
+
+        protected AzureLocation DefaultLocation => AzureLocation.WestUS;
+        protected ResourceType ResourceType => "Microsoft.Fabric/capacities";
+
+        protected FabricManagementTestBase(bool isAsync, RecordedTestMode mode)
+        : base(isAsync, mode)
+        {
+        }
+
+        protected FabricManagementTestBase(bool isAsync)
+            : base(isAsync)
+        {
+        }
+
+        [SetUp]
+        public async Task CreateCommonClient()
+        {
+            Client = GetArmClient();
+            Subscription = await Client.GetDefaultSubscriptionAsync();
+            ResourceGroup = (await Subscription.GetResourceGroupAsync(TestEnvironment.ResourceGroup)).Value;
```

Comment:
is this resource group created by the bicep file?
I am not quite sure if this could work properly during playback.
If it works fine, then it is fine.
If this does not work during playback, we need to create the resource group by our own

Reply:
You could try to playback your recording locally by change the mode to `Playback`.

Reply:
done